app/components/buttons_component.html.erb

```diff
@@ -2,6 +2,9 @@
   <div class="col-auto">
     <%= delete_button %>
   </div>
+  <div class="col-auto">
+    <%= link_to 'Cancel', :back %>
```

Comment:
What should this do if they've navigated directly to this page via an email? Then they won't have a back in their browser history to go to?

Reply:
Hmm, yes, good point.  I did some research and Rails uses the referrer if one exists, and then tries to revert to javascript's history if not.  So if you open a new tab directly to the work/collection edit page, the cancel button takes you to the blank browser window.  Assuming you are logged in and the email link gets you a new tab.

Reply:
Right, I don't think that's acceptable, especially since we're sending this link in an email. I think it needs an explicit route.

Reply:
Ok, how about this approach, which sets the back button location based on model type:

Collection --> back to dashboard
Work --> back to collection details page listing the works

Thoughts?  Also, worth doing a test for?